guides/WRITING-DOCUMENTATION.md

```diff
@@ -54,156 +68,114 @@ The description should describe what pattern does or should be used for in plain
+| **`@status [NAME]`**        | `@status Draft`                                              | Displays a status badge. `[NAME]` must be one of the following values: `Draft`, `Work in progress`, `Ready`, `Deprecated`.              |
+| **`@uswds [URL]`**          | `@uswds https://designsystem.digital.gov/components/button/` | `[URL]` is the link to the corresponding US Web Design System component.                                                                |
 
 ### Modifiers
 
 If the item you are documenting has multiple states or styles depending on added classes or pseudo-classes, you should document them in the modifiers section. [More info on modifiers from the KSS documentation site](https://warpspire.com/kss/syntax/).
 
 ### Markup
 
-Markup examples can be written in plain HTML inline or in a separate `.html` file, relative to the CSS file:
+Markup examples should be written in a separate `example.html` file, which should be located in the same directory as the SCSS docs file:
 
 ```
 Markup: Button.example.html
 ```
 
-#### Button.example.html file
-
-```HTML
-<button class="ds-c-button {{modifier}}">Button label</button>
-```
+### Style guide
 
-### Style guide: Reference
+The `reference` defines the position or URL within the site structure for a section of KSS documentation.
 
-The `reference` defines the documentation site's structure.
-
-The expected format is:
-`[Top-level slug].[Subpage slug].[Page section slug]` for example `Style guide: components.button.disabled`
+The expected format is: `[Top-level slug].[Subpage slug].[Page section slug]`.
+For example, `Style guide: components.button.disabled`.
 
 The docs site supports a maximum of 2 levels of nesting. Pages nested a 3rd level will be displayed in the body of their parent page.
 
 Pages generated from KSS comment blocks are ordered alphabetically, and page sections are displayed in the order in which they are in the SCSS file.
 
-### Documentation and guidance
+### Guidance
+
+To add a guidance section to the page the `style guide:` `[Page section slug]` should be `guidance`.
+For example: `Style guide: components.button.guidance`
 
-To add guidance content to the page, the page section's slug should be `guidance`.
-For example: `components.buttons.guidance`
+Guidance should be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`.
 
-Guidance can be written using [Markdown](https://github.com/adam-p/markdown-here/wiki/Markdown-Cheatsheet) if you include `---` right after your opening section comments, `/*`. For example
+To link to another documentation page in your guidance content use a relative url and the `{{root}}` template. This template will be replaced with your site's root url.
```

Comment:
Can we remove line 111 and put the example in the sentence. This way we can avoid using a code block for the example

```suggestion
To link to another documentation page in your guidance content use a relative url and the `{{root}}` template (i.e. `{{root}}/components/button`). This template will be replaced with your site's root url.
```